connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerSinkTaskThreadedTest.java

```diff
@@ -695,6 +706,29 @@ public Object answer() throws Throwable {
         return capturedCallback;
     }
 
+    private void expectTaskGetTopic(boolean anyTimes) {
+        final Capture<String> connectorCapture = EasyMock.newCapture();
+        final Capture<String> topicCapture = EasyMock.newCapture();
+        IExpectationSetters<TopicStatus> expect = EasyMock.expect(statusBackingStore.getTopic(
+                EasyMock.capture(connectorCapture),
+                EasyMock.capture(topicCapture)));
+        if (anyTimes) {
+            expect.andStubAnswer(() -> new TopicStatus(
+                    topicCapture.getValue(),
+                    new ConnectorTaskId(connectorCapture.getValue(), 0),
+                    Time.SYSTEM.milliseconds()));
+        } else {
+            expect.andAnswer(() -> new TopicStatus(
+                    topicCapture.getValue(),
+                    new ConnectorTaskId(connectorCapture.getValue(), 0),
+                    Time.SYSTEM.milliseconds()));
```

Comment:
WDYT about doing something like the following to reduce the duplicate code:
```
Supplier<TopicStatus> returnStatus = () -> new TopicStatus(
        topicCapture.getValue(),
        new ConnectorTaskId(connectorCapture.getValue(), 0),
        Time.SYSTEM.milliseconds());
if (anyTimes) {
    expect.AndStubAnswer(returnStatus);
} else {
    expect.AndAnswer(returnStatus);
}
```
and doing the same in the other two similar methods?

Reply:
Yeah, this was a minor change while leaving the fact there were still two other similar methods. We maybe should refactor later.